util/uvmdvgen/scoreboard.sv.tpl

```diff
@@ -64,9 +64,6 @@ class ${name}_scoreboard extends dv_base_scoreboard #(
     bit     write           = item.is_write();
     uvm_reg_addr_t csr_addr = get_normalized_addr(item.a_addr);
 
-    super.process_tl_access(item, channel);
```

Comment:
Glad to see this enabled the removal of this code duplicated in all scoreboards!